### DISCUSSION TITLE
Application approved but mail not received

### DISCUSSION THREAD

**Discussion by Saba3939** [community/community#129944] [GitHub Education]
### Hi everyone,

My educational certification was approved 3 days ago, but I have not received the certification email. What should I do?
![スクリーンショット 2024-06-27 152327](https://github.com/community/community/assets/143948435/8e999820-ac95-4e2e-88a7-6eda360a1361)

**Comment by Sworzen1** [comment]
Hello 

Look on similar [Thread](https://github.com/orgs/community/discussions/129836)

Check Spam/Junk Folder: Sometimes, emails can end up in the spam or junk folder. Make sure to check these folders in your email account.

Email Confirmation: Ensure that the email address associated with your GitHub account is correct and that you have confirmed it.

GitHub Support: You can also contact GitHub support directly through the [GitHub Support](https://support.github.com/).
